src/python_testing/TC_WEBRTC_1_3.py

```diff
@@ -0,0 +1,153 @@
+
+        webrtc.SetCallbacks(client, answer_callback, error_callback,
+                            peer_connected_callback, peer_disconnected_callback, stats_callback)
+
+        webrtc.webrtc_requestor_init()
+
+        # TODO Initialise PeerConnection without setting local offer
+
+        endpoint = self.get_endpoint(default=1)
+
+        self.step("precondition-1")
+
+        self.step("precondition-2")
+        current_sessions = await self.read_single_attribute_check_success(cluster=WebRTCTransportProvider,
+                                                                          attribute=WebRTCTransportProvider.Attributes.CurrentSessions,
+                                                                          endpoint=endpoint)
+        asserts.assert_equal(len(current_sessions), 0, "Found an existing WebRTC session")
+
+        self.step(1)
+        solicit_offer_response: WebRTCTransportProvider.Commands.SolicitOfferResponse = await self.send_single_cmd(
+            cmd=WebRTCTransportProvider.Commands.SolicitOffer(
+                streamUsage=WebRTCTransportProvider.Enums.StreamUsageEnum.kLiveView,
+                videoStreamID=NullValue,
+                audioStreamID=NullValue,
+            ), endpoint=endpoint
+        )
+        session_id = solicit_offer_response.webRTCSessionID
+        asserts.assert_true(session_id >= 0, "Invalid response")
+        asserts.assert_true(solicit_offer_response.deferredOffer, "Expected deferredOffer = True")
+
+        self.step(2)
+        offer_sessionId, remote_offer_sdp, remote_nodeid = webrtc.get_remote_offer(timeout=30)
+        asserts.assert_equal(offer_sessionId, session_id, "Invalid session id")
+        asserts.assert_true(len(remote_offer_sdp) > 0, "Invalid offer sdp received")
+
+        self.step(3)
+        webrtc.SetOffer(client, remote_offer_sdp)
+
+        self.step(4)
+        local_answer = webrtc.CreateAnswer(client)
+        await self.send_single_cmd(
+            cmd=WebRTCTransportProvider.Commands.ProvideAnswer(
+                webRTCSessionID=session_id,
+                sdp=local_answer
+            ), endpoint=endpoint
+        )
+
+        self.step(5)
+        local_candidates = webrtc.GetCandidates()
+        await self.send_single_cmd(
+            cmd=WebRTCTransportProvider.Commands.ProvideICECandidates(
+                webRTCSessionID=session_id,
+                ICECandidates=local_candidates
+            ), endpoint=endpoint
+        )
+
+        self.step(6)
+        ice_session_id, remote_candidates = webrtc.get_remote_ice_candidates(timeout=30)
+        asserts.assert_equal(ice_session_id, session_id, "Invalid session id")
+        asserts.assert_true(len(remote_candidates) > 0, "Invalid ice candidates received")
+        for candidate in remote_candidates:
+            webrtc.SetCandidate(client, candidate)
+
+        self.step(7)
+        self.wait_for_user_input("Verify WebRTC session is established")
```

Comment:
You should only do this if not running in ci; 
e.g. 
if not self.is_ci
 self.wait_for_user_input()

Reply:
Thank you for your feedback. We fixed this issue with our PR